docs/RELEASE.md

```diff
@@ -0,0 +1,16 @@
+# Release Process
+
+## QA
+
+1. Decide what commit should be used for the release and create a draft release
```

Comment:
maybe explicitly specify that only master branch commits are eligible for release

Reply:
no necessarily :P (normally it is, but hotfixes would not always be on the `master` branch for example)

I will add some info

Reply:
every release should be on the master branch. Don't we merge hofix-branches to master before releasing?